src/main/java/com/kryptnostic/conductor/rpc/EntityDataLambdas.java

```diff
@@ -0,0 +1,26 @@
+package com.kryptnostic.conductor.rpc;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.Function;
+
+public class EntityDataLambdas implements Function<ConductorSparkApi, Boolean>, Serializable {
```

Comment:
Yeah, it might require creating a ConductorPredicate wrapper. I'm going to approve for now, but filed tracking bug.